team-bayou/UNO-Shareboard
# Enhancement/#38/send confirmation code

To Test:
- Add a unv user with the user's email being your own so that you can check for the response
